GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+* Technical committee members
+
+Contributors
+++++++++++++
+
+Contributors are community members who contribute in concrete ways to the project. Anyone can become a contributor, and contributions can take many forms – not only code – as detailed in the contributing guide.
+
+Core developers
++++++++++++++++
+
+Core developers are community members who have shown that they are dedicated to the continued development of the project through ongoing engagement with the community. They have shown they can be trusted to maintain sktime with care.
+
+* **Direct access.** Being a core developer allows contributors to more easily carry on with their project related activities by giving them direct access to the project’s repository.
+* **Issue/PR management.** Core developers can review and manage issues and pull requests. This includes commenting on issues, reviewing code contributions, merging approved pull requests, and closing issues once resolved.
+* **Voting.** They can cast votes for and against merging a pull-request, and can be involved in deciding major changes to the API.
+
+New core developers can be nominated by any existing core developers. Once they have been nominated, there will be a vote by the current core developers.
+
+Voting on new core developers is one of the few activities that takes place on the project's private chat or management list. While it is expected that most votes will be unanimous, a two-thirds majority of the cast votes is enough. The vote needs to be open for at least 1 week.
+
+Core developers that have not contributed to the project (commits or GitHub comments) in the past 12 months will be asked if they want to become *emeritus core developers* and give up their direct-access, management and voting rights until they become active again.
+
+The list of core developers, active and emeritus (with dates at which they became active) is public on the sktime website.
+
+Technical committee
++++++++++++++++++++
+
+The technical committee (TC) members are core developers who have additional rights and responsibilities to avoid deadlocks and to ensure the smooth running of the project. TC members are expected to participate in strategic planning, and approve changes to the governance model.
+
+The purpose of the TC is to ensure a smooth progress from the big-picture perspective. Changes that impact the full project require a synthetic analysis and a consensus that is both explicit and informed. In cases that the core developer community (which includes the TC members) fails to reach a consensus in the required time frame, the TC is the entity to resolve the
+issue.
+
+Membership of the TC is by nomination by a core developer. A nomination will result in discussion which cannot take more than a week and then a vote by the core developers which will stay open for a week. TC membership votes are subject to a two-third majority of all cast votes as well as a simple majority approval of all the current TC members.
```

Comment:
perhaps not well formulated, I think the first sentence means that someone becomes TC member automatically after being nominated by a CD.

Also, maybe let´s be a bit more specific on what counts as a vote, how it is cast, how it is reported. Open or secret, for example? In the slack chat, or on Github? Announced in a non-public slack channel and directly to candidates, or publicly?

Reply:
oh, and are abstentions possible?